src/components/Editor/index.js

```diff
@@ -102,10 +102,21 @@ class Editor extends Component {
     if (this.props.onKeyDown) {
       this.props.onKeyDown(evt)
     }
+    
     if (evt.keyCode === 9 && !this.props.ignoreTabKey) { // Tab Key
-      document.execCommand('insertHTML', false, '&#009')
-      evt.preventDefault()
-    } else if (evt.keyCode === 13) { // Enter Key
+      if(!this.tabGuarded){
```

Comment:
You can just wrap this into the above `if` statement.

Also we can remove the `ignoreTabKey` prop now, as the new behaviour is supposed to replace it.